src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs

```diff
@@ -8,33 +8,113 @@
+        public UltimateResourceFallbackLocation Location { get { throw null; } }
     }
     [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
-    public partial class ResourceManager
+    public class ResourceManager
     {
+        protected string BaseNameField;
+        public static readonly int HeaderVersionNumber;
+        public static readonly int MagicNumber;
+        protected System.Reflection.Assembly MainAssembly;
+        protected ResourceManager() { }
         public ResourceManager(string baseName, System.Reflection.Assembly assembly) { }
+        public ResourceManager(string baseName, System.Reflection.Assembly assembly, System.Type usingResourceSet) { }
         public ResourceManager(System.Type resourceSource) { }
-        public string GetString(string name) { return default(string); }
+        public virtual string BaseName { get { throw null; } }
+        public virtual bool IgnoreCase { get { throw null; } set { } }
+        public virtual System.Type ResourceSetType { get { throw null; } }
+        protected static System.Globalization.CultureInfo GetNeutralResourcesLanguage(System.Reflection.Assembly a) { throw null; }
+        public virtual object GetObject(string name) { throw null; }
+        public virtual object GetObject(string name, System.Globalization.CultureInfo culture) { throw null; }
+        protected virtual string GetResourceFileName(System.Globalization.CultureInfo culture) { throw null; }
+        public virtual ResourceSet GetResourceSet(System.Globalization.CultureInfo culture, bool createIfNotExists, bool tryParents) { throw null; }
+        protected static System.Version GetSatelliteContractVersion(System.Reflection.Assembly a) { throw null; }
+        public System.IO.UnmanagedMemoryStream GetStream(string name) { throw null; }
+        public System.IO.UnmanagedMemoryStream GetStream(string name, System.Globalization.CultureInfo culture) { throw null; }
+        public virtual string GetString(string name) { return default(string); }
         public virtual string GetString(string name, System.Globalization.CultureInfo culture) { return default(string); }
+        protected virtual ResourceSet InternalGetResourceSet(System.Globalization.CultureInfo culture, bool createIfNotExists, bool tryParents) { throw null; }
+        public virtual void ReleaseAllResources() { }
+        // [System.Obsolete("call InternalGetResourceSet instead")]
+        // protected System.Collections.Hashtable ResourceSets;
+        protected UltimateResourceFallbackLocation FallbackLocation { get { throw null; } set { } }
+        public static ResourceManager CreateFileBasedResourceManager(string baseName, string resourceDir, System.Type usingResourceSet) { throw null; }
+        
+    }
+    [System.Serializable]
+    public class ResourceSet : System.IDisposable, System.Collections.IEnumerable
+    {
+        protected IResourceReader Reader;
```

Comment:
for consistency I removed all the protected fields. https://github.com/dotnet/standard/commit/6b4f9b5e55e902e2866e86c2f456b432acd4904a